governor/src/state/in_memory.rs

```diff
@@ -59,6 +86,16 @@ impl StateStore for InMemoryState {
     {
         self.measure_and_replace_one(f)
     }
+
+    fn measure_and_peek<T, F, E>(&self, _key: &Self::Key, f: F) -> Option<Result<T, E>>
+    where
+        F: Fn(Option<Nanos>) -> Result<(T, Nanos), E>,
+    {
+        Some(self.measure_and_peek_one(f))
+    }
+    fn reset(&self, _key: &Self::Key) {
+        self.0.store(0, Ordering::Release);
+    }
```

Comment:
Same on coverage of the `reset` function - I think it'd be great if this could be called in tests at least (: